### DISCUSSION TITLE
CSS broken after migration to Tailwind

### DISCUSSION THREAD

**Discussion by ivnsch** [ReactTooltip/react-tooltip#916] [General]
The tooltips were looking fine, then I migrated to Tailwind and now I don't see a background. Is it not compatible, or what do I've to do?

**Comment by gabrieljablonski** [comment]
Simply using Tailwind shouldn't be an issue. Are you sure you're not just missing `import 'react-tooltip/dist/react-tooltip.css'`?

**Comment by ivnsch** [comment]
The css was not found, but I upgraded to 5.7 and did that and it's working now (was using 4 before). Thanks.
